components/UserProfile/UserProfile.jsx

```diff
@@ -26,16 +26,15 @@ export default class UserProfile extends React.Component {
     ApiClient.get(`/users/${user._id}/profilePicture`, {}).then(pic => {
       this.setState({ profilePicture: pic });
     }).catch(error => {
-      console.log(error);
+      console.error(error);
     });
   }
 
   onClose = () => {
     const { onClose } = this.props;
-
-    this.state = {
+    this.setState({
```

Comment:
Is this what was causing the profy pics to show the wrong person for a brief moment?

Reply:
I evidently wasn't sober when I wrote this code, mutating state directly like a nub >.<

Reply:
@picklechips yep, because this line wasn't actually setting the profile picture to null.